perfkitbenchmarker/packages/sysbench05plus.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Module containing sysbench 0.5 (and later) installation and cleanup logic.
```

Comment:
The binaries, man files, etc are what I'm referring to. Can you install it to `vm_util.VM_TMP_DIR` instead? (`--prefix=/tmp/pkb/sysbench` or equivalent)

It seems like poor hygiene to leave the binaries, but uninstall the dynamic libs they depend on :)